### PR TITLE
Change rootUrl to rootURL

### PR DESCRIPTION
**Pull request by pritesh**

### CODE REVIEW

**Comment by cgilmour** [comment]
Looks good.

File: kubernetes/romana.rc [hunk 1]
```diff
@@ -3,13 +3,13 @@ select 0
 stuff "/bin/root -c /home/ubuntu/romana.conf^M"
 screen -t romana-topology
 select 1
-stuff "/bin/topology -rootUrl http://__MASTER_IP__:9600^M"
+stuff "/bin/topology -rootURL http://__MASTER_IP__:9600^M"
```
Comment (debedb) [inline kubernetes/romana.rc]: What's with Ctrl-M by the way?

Comment (pritesh) [inline kubernetes/romana.rc]: I think the file was initially created in win/dos env, sending in updated patch.

Comment (cgilmour) [inline kubernetes/romana.rc]: `stuff` puts things into screen's input buffer (as if a user typed it). `^M` is the carriage return.

Comment (debedb) [inline kubernetes/romana.rc]: Learn something new every day.

Comment (pritesh) [inline kubernetes/romana.rc]: yep same here, so its intentional :)

**Pull request merged**